src/gengowatcher/ui_textual.py

```
@@ -1355,11 +1039,54 @@ def _refresh_widget(self, selector_or_type, method_name: str) -> None:
+                job_id = str(job.get("id", "N/A"))[:8]
+                pair = job.get("lang_pair", "??→??")
+                words = str(job.get("word_count", job.get("words", 0)))
+                reward = f"${job.get('reward', 0):.2f}"
+                source = job.get("source", "N/A")
+                timestamp = job.get("timestamp", "")
+                if timestamp:
+                    # Truncate timestamp to just time
+                    try:
+                        ts = timestamp.split()[1] if " " in timestamp else timestamp
+                        timestamp = ts[:8]  # HH:MM:SS
+                    except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)